addons/addon-base-raas/packages/base-raas-post-deployment/lib/steps/create-service-catalog-portfolio.js

```diff
@@ -290,7 +292,7 @@ class CreateServiceCatalogPortfolio extends Service {
     const params = {
       Parameters: {
         Info: {
-          LoadTemplateFromURL: `https://${s3BucketName}.s3.amazonaws.com/service-catalog-products/${productFileName}.cfn.yml`,
+          LoadTemplateFromURL: `https://${s3BucketName}.s3.${awsRegion}.amazonaws.com/service-catalog-products/${productFileName}.cfn.yml`,
```

Comment:
suffix is hardcode, it would not work in China regions

Reply:
Done